docs/developer-guide/running-basemaps-locally.md

```diff
@@ -0,0 +1,89 @@
+# Running Basemaps Locally
+
+This guide walks you through setting up and running a local instance of the **Basemaps** system on your machine. This guide shows you how to:
+
+- Build the packages in the **basemaps** repository
+- Run the **basemaps/server** package on your machine using either:
+    - [LINZ's imagery](#using-linzs-imagery)
+    - [your own imagery](#using-your-own-imagery)
+
+
+If you're planning to contribute to the **Basemaps** repository, make sure to review the [**CONTRIBUTING**][contributing] document.
+
+## Prerequisites
+
+### NodeJS
+
+You need **[Node.js](https://nodejs.org)** version `18` or higher to build the **Basemaps** packages. You can use tools like [**fnm**](https://github.com/Schniz/fnm) or [**n**](https://github.com/tj/n) to manage **Node.js** versions on your machine.
+
+### Imagery
+
+- #### Using LINZ's Imagery
+
+    If you wish to use LINZ's imagery, you'll need **AWS credentials** with permission to read files from the **LINZ AWS S3** bucket at `s3://linz-basemaps`. These credentials will allow you to access and/or generate the bundled configuration file required for running the **Basemaps/server** package.
+
+- #### Using your own Imagery
+
+    If you wish to use your own imagery, feel free to organise your collection(s) of GeoTIFF files however you like. However, be mindful when attempting to server datasets with differing ground sampling distances (GSDs).
```

Comment:
```suggestion
    If you wish to use your own imagery, feel free to organise your collection(s) of GeoTIFF files however you like. However, be mindful when attempting to serve datasets with differing ground sampling distances (GSDs).
```

could explain why to be mindful? each folder needs to have consistent GSD/projection/XX?